crypto/pipe.go

```diff
@@ -45,7 +42,7 @@ func (c *encoder) EncryptOnFly(fileCh chan<- []byte, wg *sync.WaitGroup) error {
 					return err
 				}
 			}
-			encBuf, err := c.EnCipher.Encrypt(fileBuf, n)
+			encBuf, err := c.EnCipher.Encrypt(fileBuf)
```

Comment:
Can you elaborate on why n is not needed?

Reply:
This argument is not used.